tests/baselines/reference/arrayFrom.types

```diff
@@ -29,7 +29,7 @@ const inputALike: ArrayLike<A> = { length: 0 };
 const inputARand = getEither(inputA, inputALike);
 >inputARand : ArrayLike<A> | Iterable<A>
 >getEither(inputA, inputALike) : ArrayLike<A> | Iterable<A>
->getEither : <T>(in1: Iterable<T>, in2: ArrayLike<T>) => Iterable<T> | ArrayLike<T>
+>getEither : <T>(in1: Iterable<T>, in2: ArrayLike<T>) => ArrayLike<T> | Iterable<T>

```

Comment:
What’s this about?

Reply:
Its likely due to the fact that we now compare a signature with a trailing void that we would have previously skipped because the `minArgumentCount` didn't match, and as a result we encounter `ArrayLike` during type checking earlier than we do `Iterable`. As a result, `ArrayLike` ends up with an earlier type id and thus its order in the union changes.